plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/internal/PlayPublishTaskBase.kt

```diff
@@ -10,11 +10,14 @@ import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.Nested
 import org.gradle.internal.logging.progress.ProgressLogger
 import org.gradle.internal.logging.progress.ProgressLoggerFactory
+import org.gradle.kotlin.dsl.support.serviceOf
+import org.gradle.workers.WorkerExecutor
 
 abstract class PlayPublishTaskBase : DefaultTask(), ExtensionOptions {
     @get:Nested override lateinit var extension: PlayPublisherExtension
     @get:Internal internal lateinit var variant: ApplicationVariant
 
+    @get:Internal protected val workerExecutor = project.serviceOf<WorkerExecutor>()
```

Comment:
Do we need this property in our base task? The bootstrap task seems to be the only one using it

Reply:
At the moment, yes. I was thinking we might want to use it for future tasks though. I don't mind either way so if you want me to move it down, I'll be happy to do so.

Reply:
Let's move it into the Bootstrap task for now

Reply:
Done